tests/pom.xml

```diff
@@ -33,7 +33,6 @@
   <name>Apache Pulsar :: Tests</name>
   <modules>
     <module>docker-images</module>
-    <module>integration-tests-utils</module>
     <module>integration-tests-topologies</module>
```

Comment:
better to compile these integration modules into one. since there is no needed to separate them anymore. that would reduce pulsar's build time.

also please update tests README https://github.com/apache/incubator-pulsar/blob/master/tests/README.md